src/us/kbase/common/utils/CondorUtils.java

```diff
@@ -49,7 +49,7 @@ private static File createCondorSubmitFile(String ujsJobId, String token, String
         csf.add("batch_name = " + ujsJobId);
         csf.add("queue 1");
 
-        File submitFile = new File(String.format("%s/%s/submitscript.sub", baseDir, ujsJobId));
+        File submitFile = new File(String.format("%s.sub", ujsJobId));
```

Comment:
No longer uses /mnt/ because that is not available. It will just dump the sub file into the present directory and then delete it upon submission.